src/components/SelectFeesStrategy.js

```diff
@@ -0,0 +1,202 @@
+  getAccountUnit,
+  getAccountCurrency,
+} from "@ledgerhq/live-common/lib/account";
+import { useTheme } from "@react-navigation/native";
+import type { Account, AccountLike } from "@ledgerhq/live-common/lib/types";
+import type { Transaction } from "@ledgerhq/live-common/lib/families/ethereum/types";
+import LText from "./LText";
+import SummaryRow from "../screens/SendFunds/SummaryRow";
+import CheckBox from "./CheckBox";
+import CounterValue from "./CounterValue";
+import CurrencyUnitValue from "./CurrencyUnitValue";
+
+import SectionSeparator from "./SectionSeparator";
+import BottomModal from "./BottomModal";
+import Info from "../icons/Info";
+import NetworkFeeInfo from "./NetworkFeeInfo";
+
+type Props = {
+  strategies: any,
+  account: AccountLike,
+  parentAccount: ?Account,
+  transaction: Transaction,
+  onStrategySelect: Function,
+  onCustomFeesPress: Function,
+  forceUnitLabel?: *,
+};
+
+const CVWrapper = ({ children }: { children: * }) => (
+  <LText semiBold color="grey">
+    {children}
+  </LText>
+);
+
+export default function SelectFeesStrategy({
+  strategies,
+  account,
+  transaction,
+  onStrategySelect,
+  onCustomFeesPress,
+  forceUnitLabel,
+}: Props) {
+  const { colors } = useTheme();
+  const mainAccount = getMainAccount(account, undefined);
+  const currency = getAccountCurrency(mainAccount);
+  const unit = getAccountUnit(account);
+
+  const [isNetworkFeeHelpOpened, setNetworkFeeHelpOpened] = useState(false);
+  const toggleNetworkFeeHelpModal = useCallback(
+    () => setNetworkFeeHelpOpened(!isNetworkFeeHelpOpened),
+    [isNetworkFeeHelpOpened],
+  );
+  const closeNetworkFeeHelpModal = () => setNetworkFeeHelpOpened(false);
+
+  const isChecked = (label, transaction) => transaction.feesStrategy === label;
+
+  return (
+    <>
+      <BottomModal
+        id="NetworkFee"
+        isOpened={isNetworkFeeHelpOpened}
+        preventBackdropClick={false}
+        onClose={closeNetworkFeeHelpModal}
+      >
+        <NetworkFeeInfo onClose={closeNetworkFeeHelpModal} />
+      </BottomModal>
+
+      <View>
+        <SectionSeparator lineColor={colors.lightFog} />
+        <SummaryRow
+          onPress={toggleNetworkFeeHelpModal}
+          title={<Trans i18nKey="send.summary.fees" />}
+          additionalInfo={
+            <View>
+              <Info size={12} color={colors.grey} />
+            </View>
+          }
+        >
+          {null}
+        </SummaryRow>
+        <ScrollView style={styles.strategiesContainer}>
+          {strategies.map((s, index) => (
+            <TouchableOpacity
+              key={index + s.label}
+              onPress={() => {
+                onStrategySelect({
+                  amount: s.amount,
+                  label: s.label,
+                  userGasLimit: s.userGasLimit,
+                });
+              }}
+              style={[
+                styles.feeButton,
+                {
+                  borderColor: isChecked(s.label, transaction)
```

Comment:
having function calls directly in template on react-native can slow down the app quite a lot, since the component you're looping on here is not so small maybe extract it all into a function Component :thinking: 
+ you repeat this isChecked several times